test/cbmc/proofs/freeFileContextMem/freeFileContextMem_harness.c

```diff
@@ -34,6 +34,8 @@
 extern OtaAgentContext_t otaAgent;
 extern void freeFileContextMem( OtaFileContext_t * const pFileContext );
 
+__CPROVER_bool nondet_bool();
```

Comment:
Why does this need to be declared here? And why the special return type given the earlier example `nondet_int` was returning a standard type?

Reply:
Here, this is only used for a true/false decision. We could, however, also pick a different type (that can be converted to an `int`) in here.